ui/redux/selectors/subscriptions.js

```diff
@@ -26,6 +26,16 @@ export const selectSubscriptionUris = createSelector(
   (subscriptions) => subscriptions && subscriptions.map((sub) => sub.uri)
 );
 
+export const selectSubscriptionIds = createSelector(selectSubscriptions, (subscriptions) => {
+  if (subscriptions) {
+    const subscriptionIds = subscriptions.map((sub) => parseURI(sub.uri).channelClaimId);
+    // assert(subscriptionIds.every((x) => x && x.length === 40));
```

Comment:
this assert is commented out, but possible to instead do it once on writing to store?